.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Information:**
+- Prism version: [e.g. 1.14, latest from the download page, etc.]
+- Environment: [e.g. Browser, Node, Webpack]
+
+Does the latest version of Prism from the [download page](https://prismjs.com/download.html) also have this issue?
+Do Prism's [usage section](https://prismjs.com/index.html#basic-usage) or [docs](https://prismjs.com/extending.html) say anything about the issue?
```

Comment:
I might add a line here saying "If you're using Prism with Node or Webpack, please see the usage section." Since that's an issue we get commonly, it's worth calling out more explicitly.

Reply:
Should I also remove the current line of text linking to the usage section?

Reply:
Maybe make it more generic like, "Does Prism's documentation say anything about the issue?" and just link to the website.